app/controllers/govspeak_controller.rb

```diff
@@ -0,0 +1,10 @@
+class GovspeakController < ApplicationController
+  skip_before_action :verify_authenticity_token
```

Comment:
we need to make sure only authenticated users can use it

  before_action :authenticate_user!

Reply:
check